.eslintrc

```diff
@@ -5,5 +5,6 @@
 
 	"rules": {
 		"id-length": [2, { "min": 1, "max": 27 }],
-	},
+		"no-magic-numbers": ["warn", { "ignore": [0, 1, -1] }]
+	}
```

Comment:
```suggestion
	},
```

Reply:
If I add the trailing comma, eslint itself complains.

Reply:
eslint doesn't run on `.eslintrc`, and whatever's complaining is wrong because eslintrc isn't json, it's JSONC.

Reply:
> eslint doesn't run on `.eslintrc`, and whatever's complaining is wrong because eslintrc isn't json, it's JSONC.

My bad, the warning was coming from another VS Code extension.